src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exception;
+
+public class DukeException extends Exception {
+    public DukeException(String s) {
```

Comment:
I think it would be more informative to name the string as `errorMessage`, so that it would be easier for others to understand the purpose of the string on first glance.